package.json

```diff
@@ -113,8 +110,10 @@
     "apollo-cache-inmemory": "1.2.10",
     "apollo-client": "2.4.7",
     "apollo-link": "1.2.1",
-    "babel-core": "6.26.3",
+    "babel-core": "7.0.0-bridge.0",
     "babel-jest": "23.6.0",
+    "babel-plugin-annotate-pure-calls": "0.4.0",
+    "babel-plugin-dev-expression": "0.2.1",
```

Comment:
`babel-plugin-dev-expression` doesn't appear to be used in this project, why add it?

Reply:
It's used in rollup as a bundling posttranspilation step afaik

Reply:
It is used in the babelConfig to cut out the invariant messages in prod mode

Reply:
Those `invariant` messages are all found inside the various React dependencies, though, right?

Reply:
And in the react components we are giving where we check for !!client etc